projects/max14906/main.c

```diff
@@ -0,0 +1,322 @@
+#include <stdio.h>
```

Comment:
You should add file description, author, copyright, and license.

Reply:
Adding.